src/presentation/ui/component/structural/infoDialog.templ

```diff
@@ -0,0 +1,7 @@
+package componentStructural
+
+templ InfoDialog(message, bgColor string) {
```

Comment:
Dialog means a modal with a yes or no button. This seems to be an alert, no?
https://flowbite.com/docs/components/alerts/

Reply:
This component has been removed cause we no longer use it.